### DISCUSSION TITLE
Error building : An error occurred while performing transpilation: The file “Resources” couldn’t be saved in the folder “module”.

### DISCUSSION THREAD

**Discussion by tom-eager** [skiptools/skip#218] [Q&A] [answered]
I think I removed all of my compile error but ended up with one error left. 

 `An error occurred while performing transpilation: The file “Resources” couldn’t be saved in the folder “module”.`

I'm not sure what to do to address this issue.  Hints ?

**Comment by marcprux** [comment]
Does your project have a `Resources/` folder under the `Sources/` for the module? If so, does it contain anything?

If you've changed anything structurally in your project, you might try just deleting your `~/Library/Developer/Xcode/DerivedData/` folder and restarting Xcode, which sometimes helps.

**Comment by tom-eager** [comment]
Yes, There is a folder `Resources/`  under the `Sources/` for the module.  This folder contains the two project generated files: Localizable.xcstrings and Module.xcassets.  I have not changed them.    I deleted the derivedData folder for this project. Quit xCode and then started it up again.  I still have the issue after I refreshed my Packages.

**Comment by marcprux** [comment]
How odd. Did the project ever build and run for you? If you create and run a fresh new project with `skip init --open-xcode --appid=skip.hello hello-skip HelloSkip`, do you still get the same error?

**Reply by tom-eager** [reply]
I created a fresh project, ran it (skipper with the beating heart).  There was no issue.  I added my swift files, cleaned it up like the blog example. I have never been able to see my code work.

**Reply by aabewhite** [reply]
Did you make any changes to Package.swift?

I don't know if this is sensitive code - is it something you could send to support@skip.tools so we could build it ourselves?

**Comment by ptrkstr** [comment, marked as answer]
@tom-eager, @aabewhite, @marcprux This happened to me too. It occurred when a README.md was placed in the `Package > Sources > Project` folder. Removing this file fixes the problem.

<img src="https://github.com/user-attachments/assets/562cedbb-4331-47b8-9c6c-8a4df781ce64" height=300>

**Reply by tom-eager** [reply]
I had included a file in the` Package > Sources > Project` folder. Removing this file fixes the problem.  My file was not a swift file but an `.mp4`

**Reply by marcprux** [reply]
Thanks for letting us know. That's definitely unexpected behavior.
